distro/superbuild/cmake/externals.cmake

```diff
@@ -224,9 +242,18 @@ endif()
 
 ###############################################################################
 # PythonQt
+set(PythonQt_REPO)
+set(PythonQt_TAG)
+if(${DD_QT_VERSION} VERSION_GREATER "4")
+  set(PythonQt_REPO https://github.com/patmarion/PythonQt.git)
+  set(PythonQt_TAG fix-compile-error-on-qt5.8)
```

Comment:
Okay... but at least commontk/PythonQt#50 would let us use the same *repository* in either case, and just a different branch.